src/account.c

```diff
@@ -1241,10 +1241,58 @@ int account_debug(struct re_printf *pf, const struct account *acc)
 		}
 		err |= re_hprintf(pf, "\n");
 	}
-	err |= re_hprintf(pf, " call_transfer:         %s\n",
+	err |= re_hprintf(pf, " call_transfer:%s\n",
 			  account_call_transfer(acc));
-	err |= re_hprintf(pf, " extra:         %s\n",
+	err |= re_hprintf(pf, " extra:        %s\n",
 			  acc->extra ? acc->extra : "none");
 
 	return err;
 }
+
+
+/**
+ * Print the account information in JSON
+ *
+ * @param od  Account dict
+ * @param odcfg  Configuration dict
+ * @param acc User-Agent account
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int account_json_api(struct odict *od, struct odict *odcfg, const struct account *acc)
+{
+	int err = 0;
+
+	if (!acc)
+		return 0;
+
+	/* account */
+	err |= odict_entry_add(od, "aor", ODICT_STRING, acc->aor);
+	err |= odict_entry_add(od, "display_name", ODICT_STRING, acc->dispname);
```

Comment:
acc->dispname can be NULL, so please add a conditional for this line:

```c
if (acc->dispname)
```

Reply:
Cool! Added some other null check ones as well.